README.md

```diff
@@ -28,7 +28,13 @@ npm run develop
 
 Your site is running on `http://localhost:8000` and GraphQL tool helper on `http://localhost:8000/___graphql`.
 
-Every master commits will be published automatically to https://pedantic-edison-3897de.netlify.com/. Thanks to [Netlify](https://www.netlify.com/).
+### Environments
+| Environment        | Description | 
+| :----------: |:-------------:| 
+| Dev | Each PR will be built and hosted on Netlify. |
```

Comment:
> Each PR will be built and hosted on Netlify.

How is that for dev?

Reply:
This is what I mean for dev environments:
<img width="706" alt="image" src="https://user-images.githubusercontent.com/1214890/68527532-1d920b00-02e8-11ea-8d8a-5a67cd6f6841.png">

Reply:
Oh, I just learned that Netlify builds on PR too.